packages/back-end/src/app.ts

```diff
@@ -336,6 +336,7 @@ app.get("/ideas/recent/:num", ideasController.getRecentIdeas);
 // Metrics
 app.get("/metrics", metricsController.getMetrics);
 app.post("/metrics", metricsController.postMetrics);
+app.post("/metrics/auto-metrics", metricsController.postAutoGeneratedMetrics);
```

Comment:
I think I should consolidate this with the existing post endpoint for getting the list of metrics we think we can create

Reply:
Its kinda weird to have two post endpoints, but one lives on the datasource controller and the other lives within the metrics controller.